README.md

```diff
@@ -1 +1,18 @@
-# talks
\ No newline at end of file
+# Flutter Jakarta Talks
+
+We want Flutter Jakarta to be a place for flutter enthusiasts to share knowledge about development with Flutter. This repository is intended as an archive and talks proposal for anyone who wants to be a speaker at our monthly meetup. We usually hold meetups in the last week every 1 month (if possible).
+
+## Submit Your Talks Proposal
+
+Please submit your talk proposal through github issues, you have 30 minutes to bring the talk you want. Make sure your talk proposal includes the following information:
+
+- Talk Title
+- Target Audience
+- Abstract
+- Outline
+- Speaker Bio
+- Contact
+
+## How Talks Are Selected
+
+There will be 2 speakers in each meetup, we will choose talks based on a theme or similar target audience. We want this community to be a warm place for you for sharing. We will contact you via github or alternative contacts you have informed.
```

Comment:
`There will be 2 speakers in each meetup, and we will chose talks based on a theme or a similar target audience. We want this community to be a warm and comfortable place for you to share. We will contact you via GitHub or alternative information you have provided.`

Reply:
> @dehamzah I've already had a look, there's just a couple of changes. Also I was wondering if you were planning to upload the previous 2 meetups' content?
> 
> Also will let you know later if I think of more things to put in here. Thanks!

I also planning to upload our previous meetups content here, but i was thinking to separate the repo for talks and workshop, so there will be one more repo like this but for workshops content.